components/list/index.ts

```diff
@@ -0,0 +1,82 @@
+import m from "mithril";
+import getParentByClassname from "./getParentByClassname";
+import { THEME_DEFAULT } from "../types";
+
+
+export type Item = {
+    /** optional css-class for individual item */
+    class?: string;
+    [p: string]: any;
+};
+
+
+export type Attrs = {
+    /** list of items to display */
+    items: Array<Item>;
+    /** property to use as id, must point to a string-value. Defaults to "value" */
```

Comment:
tatsächlich... Defaults to "name"

Reply:
Thx. Fixed by setting the default property to `value`